src/compiler/scanner.ts

```diff
@@ -2456,7 +2457,7 @@ export function createScanner(languageVersion: ScriptTarget, skipTrivia: boolean
                 }
                 else {
                     regExpFlags |= flag;
-                    const availableFrom = regExpFlagToFirstAvailableLanguageVersion.get(flag)!;
+                    const availableFrom = regExpFlagToFirstAvailableLanguageVersion.get(flag)! as unknown as ScriptTarget;

```

Comment:
This is... awkward.

Reply:
Nit: the non-null assertion is not needed anymore